openmc/mixin.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+
+class Equality(object):
```

Comment:
I think it would be good to be explicit about this being a mixin by calling it `EqualityMixin`